romochka_bembot.js

```diff
@@ -56,11 +59,12 @@ const generateBemhtmlLink = bemhtml => {
  * @returns {String} Link
  */
 const generateBemjsonLink = bemjson => {
-	const trimedBemjson = _.trimEnd(bemjson, ';');
+	assert(typeof bemjson === 'string', 'Bemjson must be a string');
+
+	const trimedBemjson = bemjson.replace(/;+$/, '');
```

Comment:
`bemjson.trim().splice(0, -1)` in this case it's not a good choose because `;` may be in the string but also may be not.